positional/tests/parse_row.rs

```diff
@@ -35,3 +35,34 @@ fn ser_de() {
         FromPositionalRow::from_positional_row(&row).expect("error converting from positional row");
     assert_eq!(original_data, data);
 }
+
+#[test]
+fn empty_string() {
+    let row = <Data as FromPositionalRow>::from_positional_row("");
+    assert_eq!(
+        row.err().map(|e| e.to_string()),
+        Some(
+            "The row passed is too small to work with the fields definition. The row needs to have at least 30 unicode chars. Passed row is: `` with length 0"
+                .to_string()
+        )
+    );
+}
+
+#[test]
+fn string_smaller_than_field_definition() {
+    let row_content = "1    ---10";
+    let row = Data::from_positional_row(row_content);
+    assert_eq!(
+        row.err().map(|e| e.to_string()),
+        Some(
+            format!("The row passed is too small to work with the fields definition. The row needs to have at least 30 unicode chars. Passed row is: `{}` with length {}", row_content, row_content.len())
+        )
+    );
+}
+
+#[test]
+fn unicode_chars() {
+    let row = Data::from_positional_row("Noël ---10the address is this ")
+        .expect("error converting from positional row");
+    assert_eq!(Data::new("Noël", 10, "the address is this"), row);
```

Comment:
Does this fail with the previous implementation?

Reply:
It used to, I edited my comment. I did `start..end` instead of `start..=end`

Reply:
Just want to make sure the test fails before your commit and works with your commit, so we're sure we're testing the right thing!

Reply:
yes! On master this fails:
`thread 'unicode_chars' panicked at 'error converting from positional row: ParseIntError { kind: InvalidDigit }', positional/tests/parse_row.rs:42:10`

It takes the first 5 digits, resulting in `Noël` without the ending space.
and for the next 5 as age, it gets ` ---1` with an initial space. That fails to be parsed as an i32.

Reply:
Awesome, thank you!